src/lib/Support/SafeNames.cpp

```diff
@@ -200,6 +202,17 @@ class SafeNames::Impl
                     return getReserved(t);
                 }
 
+                if constexpr(T::isAlias())
+                {
+                    MRDOCS_ASSERT(! t.Name.empty());
+                    return t.Name;
+                }
+
+                if constexpr(T::isUsing()) {
+                    MRDOCS_ASSERT(! t.Name.empty());
+                    return getReserved(t);
```

Comment:
you should only use the reserved name for a `UsingInfo` if it's a using directive (i.e. `using namespace`)